e2e-tests/portal/cypress/support/utils/insertElement.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * insertElement
+ * inserts an element in a div
+ *
+ * @param {String} divId
+ */
+const insertElement = (divId) => {
```

Comment:
Do we need to add `divId` void check?

Reply:
will never be void as passing manually to it